airflow/www/templates/airflow/dag.html

```diff
@@ -206,9 +207,6 @@ <h4 class="pull-right js-dataset-triggered" style="user-select: none;-moz-user-s
           <li><a href="{{ url_for('Airflow.gantt', dag_id=dag.dag_id, root=root, num_runs=num_runs_arg, base_date=base_date_arg, execution_date=execution_date_arg) }}">
             <span class="material-icons" aria-hidden="true">vertical_distribute</span>
             Gantt</a></li>
-          <li><a href="{{ url_for('Airflow.dag_details', dag_id=dag.dag_id) }}">
```

Comment:
For now, let's leave this and just change the URL. I think it will be best to switch all of these buttons over at once.

Reply:
Assuming here also you are referring to change the URL to `{{ url_for('Airflow.grid', dag_id=dag.dag_id, num_runs=num_runs) }}` will create redundant buttons and will look like below
![image](https://github.com/apache/airflow/assets/35839624/47714e53-650e-46d1-93ff-39fddffcc7b2)

I think it's best removed. We can keep it in `dags.html` anyways. Let me know if u meant something else by any change.